src/main/Core/BrowserWindow/BrowserWindowModule.ts

```diff
@@ -178,6 +183,13 @@ export class BrowserWindowModule {
 
         ipcMain.on("escapePressed", () => shouldHideWindowOnEscapePressed() && browserWindowToggler.hide());
 
+        app.on(
+            "second-instance",
+            () =>
+                shouldToggleWindowOnSecondInstance() &&
```

Comment:
Should this really be configurable? Why shouldn't we just always show/focus the already running instance when a second instance is launched?

Reply:
I made it an option in case anyone who doesn't need this feature wants ueli to just silently quit the second instance the way it currently does. I can remove it if you'd like

Reply:
Yes, please remove it.